README.md

```diff
@@ -1254,10 +1254,17 @@ Mode:
         Background: "blue"
 ```
 
+> **Note**: `StyleOverStrike` and `StyleOverLine` must be set at the top level and cannot be specified in the `Mode` section.
```

Comment:
For record, you can use 


```suggestion
> [!NOTE]
> `StyleOverStrike` and `StyleOverLine` must be set at the top level and cannot be specified in the `Mode` section.
```

The rendering will be better

Reply:
That's right. Thank you.